scripts/apps/authoring/authoring/directives/ItemCarouselDirective.js

```diff
@@ -41,10 +42,15 @@ export function ItemCarouselDirective($timeout) {
              * otherwise carousel height is messed up
              */
             scope.$watch('items', (items) => {
-                if (!items) {
+                const itemsString = JSON.stringify(items);
```

Comment:
you can use [angular.toJson](https://docs.angularjs.org/api/ng/function/angular.toJson) which would remove all angular added properties (if any), not sure if that's what @tomaskikutis was talking about, if not then ignore this

Reply:
@petrjasek oh I didn't know about that, thanks.